### PR TITLE
Use @digitalbazaar/vc-status-list v4.0

### PR DESCRIPTION
**Pull request by JSAssassin**
addresses #87

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # bedrock-vc-issuer ChangeLog
 
+## 18.0.0 - TBD
+
+### Changed
+- **BREAKING**: Use `@digitalbazaar/vc-status-list` v4.0, now includes
+`statusPurpose` in SLC VCs.
```
Comment (dlongley) [inline CHANGELOG.md:7]:
```suggestion
  `statusPurpose` in SLC VCs.
```

**Review by dlongley — APPROVED** [review]
Waiting on @davidlehn to comment on re-exported constants (that are used here).

File: CHANGELOG.md (hunk 1 above)
Comment (JSAssassin) [reply CHANGELOG.md]: Fixed [fea601f](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/88/commits/fea601f182633ca8cee2383b13fb06f97c2343aa)

**Comment by dlongley** [comment]
@JSAssassin I merged and released the related bedrock context packages so this needs an update for those.

**Comment by JSAssassin** [comment]
@dlongley bedrock context packages updated.

**Pull request merged**